rsocket-core/src/main/java/io/rsocket/frame/FrameUtil.java

```diff
@@ -0,0 +1,108 @@
+package io.rsocket.frame;
+
+import io.netty.buffer.ByteBuf;
+import io.netty.buffer.ByteBufUtil;
+import io.netty.buffer.Unpooled;
+
+public class FrameUtil {
+  private FrameUtil() {}
+
+  public static String toString(ByteBuf frame) {
+    FrameType frameType = FrameHeaderFlyweight.frameType(frame);
+    int streamId = FrameHeaderFlyweight.streamId(frame);
+    StringBuilder payload = new StringBuilder();
```

Comment:
worth presizing, 16 default will require a realloc.

Reply:
maybe? this was kind of for debugging purposes, but I can add a size